ci/test.sh

```diff
@@ -2,7 +2,7 @@
 set -x
 
 export IDEPROBE_DISPLAY=xvfb
-export PANTS_PLUGIN_PATH="/tmp/pants.zip"
+#export PANTS_PLUGIN_PATH="/tmp/pants.zip"
```

Comment:
You can actually delete this, and all the logic related to this ENV, as it will be done in a different way

Reply:
Ok